src/sentry/tasks/post_process.py

```diff
@@ -1595,33 +1595,90 @@ def check_if_flags_sent(job: PostProcessJob) -> None:
+            return
 
-    # Don't run if there's already a task in progress for this issue
-    lock_key, lock_name = get_issue_summary_lock_key(group.id)
-    lock = locks.get(lock_key, duration=1, name=lock_name)
-    if lock.locked():
-        return
+        # Don't run if there's already a task in progress for this issue
+        lock_key, lock_name = get_issue_summary_lock_key(group.id)
+        lock = locks.get(lock_key, duration=1, name=lock_name)
+        if lock.locked():
+            return
 
-    if is_seer_scanner_rate_limited(group.project, group.organization):
-        return
+        if is_seer_scanner_rate_limited(group.project, group.organization):
+            return
+
+        start_seer_automation.delay(group.id)
+    else:
+        # Triage signals V0 behaviour
+
+        # If event count < 10, only generate summary (no automation)
+        if group.times_seen < 10:
+            # Check if summary exists in cache
+            cache_key = get_issue_summary_cache_key(group.id)
+            if cache.get(cache_key) is not None:
+                return
+
+            # Early returns for eligibility checks (cheap checks first)
+            if not is_issue_eligible_for_seer_automation(group):
+                return
+            if is_seer_scanner_rate_limited(group.project, group.organization):
+                return
+
+            # Atomically set cache to prevent duplicate summary generation
+            summary_dispatch_cache_key = f"seer-summary-dispatched:{group.id}"
+            if not cache.add(summary_dispatch_cache_key, True, timeout=30):
+                return  # Another process already dispatched summary generation
+
+            # Generate summary (no automation)
+            generate_issue_summary_only.delay(group.id)
+        else:
+            # Event count >= 10: run automation
+            # Check seer_last_triggered first (long-term check to avoid re-running)
+            if group.seer_autofix_last_triggered is not None:
+                return
+
+            # Early returns for eligibility checks (cheap checks first)
+            if not is_issue_eligible_for_seer_automation(group):
+                return
+            if is_seer_scanner_rate_limited(group.project, group.organization):
+                return
 
-    start_seer_automation.delay(group.id)
+            # Atomically set cache to prevent duplicate dispatches (returns False if key exists)
+            automation_dispatch_cache_key = f"seer-automation-dispatched:{group.id}"
+            if not cache.add(automation_dispatch_cache_key, True, timeout=300):
+                return  # Another process already dispatched automation
+
+            # Check if summary exists in cache
+            cache_key = get_issue_summary_cache_key(group.id)
+            if cache.get(cache_key) is not None:
+                # Summary exists, run automation directly
+                run_automation_for_group.delay(group.id)
```

Comment:
in this case, what guards against re-triggering fixability scoring on every event which is outside the 5 minute dispatch window?

since this will happen unconditionally:

```
run_automation_for_group -> run_automation -> _generate_fixability_score -> calls seer API
```

the fixability endpoint will return the fixability score if it already exists for the issue, but it will need to hit the issue summary DB for that. seems expensive to do for every new event when we can check `group.seer_fixability_score` in sentry

Reply:
that sounds good, thanks for clarifying

Reply:
wait but `seer_autofix_last_triggered` is only set if the issue is deemed fixable-enough